src/cascadia/WpfTerminalTestNetCore/WpfTerminalTestNetCore.csproj

```diff
@@ -1,15 +1,17 @@
-<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <RuntimeFrameworkVersion>3.1.18</RuntimeFrameworkVersion>
+    <TargetFramework>net6.0-windows</TargetFramework>
+    <RuntimeFrameworkVersion>6.0.9</RuntimeFrameworkVersion>

```

Comment:
FYI, we have to force the runtime version to a specific value because we cannot use the public nuget.org feed. It is a requirement for projects compiled by Microsoft that we have full control over all our package dependencies :smile: